app/cli/run.py

```
@@ -0,0 +1,61 @@
+                on_task_event=_on_event,
+                on_flow_started=_on_started,
+            )
+        )
+    except Exception as e:  # noqa: BLE001
+        _render_run_footer(collected_events, console)
+        console.print(f"[red]flow failed:[/red] {e}")
+        fid = captured_flow_id["id"]
+        if fid:
+            console.print(f"[red]flow_id:[/red] {fid}")
+            console.print(f"[dim]→ atelier status {fid}[/dim]")
+        raise typer.Exit(code=1)
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)